The main POM is not deployed

# Failed to Resolve Dependencies for metricshub

When building a project `org.sentrysoftware.metricshub:metricshub` , the following error occurs:

```
[ERROR] Failed to execute goal on project metricshub-windows: Could not resolve dependencies for project org.sentrysoftware.metricshub:metricshub-windows:pom:0.9.01-SNAPSHOT: Failed to collect dependencies at org.sentrysoftware:metricshub-jre-windows:zip:1.0.00: Failed to read artifact descriptor for org.sentrysoftware:metricshub-jre-windows:zip:1.0.00: Failure to find org.sentrysoftware:metricshub-jre-builder:pom:1.0.00
```

--- a/pom.xml
+++ b/pom.xml
@@ -77,6 +77,16 @@
 			<modules>
 				<module>metricshub-jre-windows</module>
 			</modules>
+			<build>
+				<plugins>
+					<plugin>
+						<artifactId>maven-deploy-plugin</artifactId>
+						<configuration>
+							<skip>true</skip>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
 		</profile>
 		<profile>
 			<id>linux</id>
@@ -96,9 +106,6 @@
 		<plugins>
 			<plugin>
 				<artifactId>maven-deploy-plugin</artifactId>
-				<configuration>
-					<skip>true</skip>
-				</configuration>
 			</plugin>
 			<!-- artifact (Reproducible Build) -->
 			<plugin>

